tests/e2e/test_client.py

```diff
@@ -236,6 +236,58 @@ def test_upload_csv_success(self):
             True,
         )
 
+    @unittest.skip("This is a plus subscription endpoint.")
```

Comment:
Did you run it manually and make sure it works ?

Reply:
Many times :) 
<img width="460" alt="image" src="https://github.com/duneanalytics/dune-client/assets/28662355/84715ac4-e9cb-4d62-8e6c-6cb996ee5210">